src/components/Button/Button.tsx

```diff
@@ -40,10 +49,12 @@ const Button = ({
   return (
     <button
       type="button"
-      className={["storybook-button", `storybook-button--${size}`, mode].join(
+      className={["storybook-button",
+        // `storybook-button--${size}`,
+        mode].join(
```

Comment:
Personally gw find this line a bit not intuitive, I think its much cleaner kalo lu pake css overriding
kaya pertama bisa gini 
```css
.button-primary .button-secondary {
    color: #FFFFFF;
font-family: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
  font-weight: 700;
  border: 0;
  border-radius: 0.5rem;
  cursor: pointer;
  display: inline-block;
  line-height: 1;
  padding-left: 1.75rem;
  padding-right: 1.75rem;
  padding-top: 1.25rem;
  padding-bottom: 1.25rem;
}

.button-primary {
  background-color: rgba(17, 24, 39, 1);
}

.storybook-button--secondary {
  background-color: #FF3B3A;
}
```

ato secondly, just type the name explicitly, jangan pake join or sth (really makes it hard to read soalnya styling itu harusny g ad logic lagi)